src/test/cucumber/per_project_configuration.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Per-project configuration
+
+  Background:
+    Given the internet is reachable
+    And an initialised environment
+
+  Scenario: Env without providing an .sdkmanrc file
+    Given the system is bootstrapped
+    When I enter "sdk env"
+    Then I see ".sdkmanrc not found."
```

Comment:
This leads me to think about how we can generate one of these descriptors in the future. It would be nice to eventually do `sdk env generate` or something like that to create a basic descriptor. For now this is perfect though.

Reply:
Maybe something like `sdk env add groovy 2.5.9`?